spec/controllers/catalog_controller_spec.rb

```diff
@@ -35,6 +35,20 @@
       expect(facets["state_ssim"]).to eq ["complete", 1]
       expect(facets["display_subject_ssim"]).to eq [output.imported_metadata[0].subject.first, 1]
     end
+    it "can search by local identifiers" do
+      persister.save(resource: FactoryGirl.create_for_repository(:scanned_resource, local_identifier: "p3b593k91p"))
+
+      get :index, params: { q: "p3b593k91p" }
+      expect(assigns(:document_list).length).to eq 1
+    end
+    context 'with imported metadata' do
```

Comment:
I don't think there's such a thing as an imported local identifier. @escowles ?

Reply:
We're not mapping it now, but I think that records with local identifiers in 024 (like the Cicognara records) should have those imported into local_identifier.

Reply:
Okay. This test still isn't right though - you need something like `FactoryGirl.create_for_repository(:scanned_resource, imported_metadata: [ImportedMetadata.new(local_identifier: "p3b593k91p")])`

Reply:
Understood, I shall proceeding by using a fixture from 8543429